src/api/commands/itemimage/ItemImageRemoveCommand.ts

```diff
@@ -46,6 +46,22 @@ export class ItemImageRemoveCommand extends BaseCommand implements RpcCommandInt
         return this.itemImageService.destroy(data.params[0]);
     }
 
+    public async validate(data: RpcRequest): Promise<RpcRequest> {
+        if (data.params.length < 1) {
+            throw new MessageException('Requires arg: ItemImageId');
+        }
+        // find itemImage
+        const itemImageId = data.params[0];
+        if (!itemImageId) {
```

Comment:
this if is unnecessary, just validate that the id is number, and if not, throw InvalidParamException